Includes missing repository link on development documentation. (#273)
tatianamac/selfdefined: 11ty/documentation/development.md

We are using [Eleventy](https://www.11ty.io/) to generate the site based on the given definitions. As a module bundle we use Parcel.

> We’ve compiled a list of [contribution guidelines](https://github.com/selfdefined/web-app/blob/prod/CONTRIBUTING.md) to make it easy for you to start work on the project.

Reviewer: ```suggestion
We’ve compiled a list of [contribution guidelines](https://github.com/selfdefined/web-app/blob/prod/CONTRIBUTING.md) to help you start working on the project.
```
In general I try to avoid saying things are easy.

Reviewer 2: Agreed. That was on me/my sleepy brain when I made the placement suggestion 🙈

```

This will generate a minified CSS bundle as well as run Eleventy’s production build. Build files are located in `dist`.

## Contributing

> Follow the [contributing guidelines](https://github.com/selfdefined/web-app/blob/prod/CONTRIBUTING.md) to start contributing.

Reviewer 2: I’d like to have this section further at the top. Maybe instead of a section, we can appnd a sentence to the intro.

> We’ve compiled a list of contribution guidelines to make it easy for you to start work on the project.

Something like this, maybe?

Reviewer: I like this.

Author: Thank you for the feedback. Will make this change!
